bin/ecs-deploy

```diff
@@ -113,7 +113,7 @@ config.fetch(:one_off_commands, []).each do |one_off_command|
   task = nil
   while waiting <= wait_time do
     task = ecs.describe_tasks(tasks: [task_arn], cluster: cluster).tasks.first
-    break if task.last_status == "STOPPED"
+    break if task.nil? || task.last_status == "STOPPED"
```

Comment:
If you just want to guard against nil values, I suggest using the excellent **ampersand dot** operator:

```ruby
task&.last_status == "STOPPED"
```

```console
$ irb
> nil.foo
Traceback (most recent call last):
        2: from /Users/lukas/.rbenv/versions/2.5.1/bin/irb:11:in `<main>'
        1: from (irb):1
NoMethodError (undefined method `foo' for nil:NilClass)
> nil&.foo
=> nil
```

Reply:
Cool idea, but unfortunately it breaks the desired behavior:
```nil&.last_status == "STOPPED"``` evaluates to ```nil == "STOPPED"``` which is ```false```whereas we want ```true```  in this case